src/Controls/src/Core/Shapes/StrokeShapeTypeConverter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.ComponentModel;
+using System.Globalization;
+
+namespace Microsoft.Maui.Controls.Shapes
+{
+	public class StrokeShapeTypeConverter : TypeConverter
```

Comment:
This is really just a converter for a rounded rectangle, perhaps it should be named as such... `RoundedRectangleStrokeShapeTypeConverter`?